### PR TITLE
Add ENTE exchanges

### PR DESCRIPTION
**Pull request by systemcatch**
This parser gets real time exchange data from http://www.enteoperador.org/newsite/flash/SER.html.  There are 7 exchanges in total though the map already has HN->NI and CR->NI.

### CODE REVIEW

File: parsers/ENTE.py [hunk 1]
```diff
@@ -0,0 +1,226 @@
...
+    MX = interconnections['MX->GT']*GT['MX']
+    HON = interconnections['HN->NI']*HN['NI']
+
+    eqs = np.array([[a, b, 0], [a, 0, c], [0, b, c]])
+    res = np.array([net_production['SV'], net_production['GT']-MX, net_production['HN']-HON]) 
+
+    solution = np.linalg.solve(eqs, res)
+
+    #Factor to account for transmission losses.
+    GT['SV'] = plusminus(solution[0]+0.5)
```
Comment (corradio) [inline parsers/ENTE.py]: Is the 0.5 an arbitrary number?

**Comment by systemcatch** [comment]
So far the system of equations is giving good results.  I'll check the determinant of the matrix to make sure though.

**Comment by corradio** [comment]
A good solution I've used before for those kind of problems is to add a L2
regularisation term on the flows to make sure we get the smallest flows
that satisfy the equations.
Can you, in the header of the file, make it very clear which flows are
estimated and which one are measured?
I'd actually be reluctant to add the estimated flows, as the Electricity
Map is all about showing data coming from other sources instead of trying
to be too clever ourselves.

I'll think a bit more about it though.

Olivier

**Comment by systemcatch** [comment]
I don't know what a L2 regularization term is but I'll do some reading on it.  I take your point on being too clever though, we don't want to add more uncertainty to the data! 

Maybe the best thing to do for now is to tweak the parser to return only the 4 flows that are safe and leave out the 3 flows involved in the simultaneous equations.  Once I go through the maths properly we can either add them back in or scrap them if the equations are wrong/uncertain.

**Comment by corradio** [comment]
Yes indeed I think that's safer for now!

**Comment by corradio** [comment]
Let me know when I should review/merge

**Comment by systemcatch** [comment]
It's good to go.  All the unsafe flows are set to zero now.

**Comment by corradio** [comment]
Great, can you update the exchanges.json files to add the coordinates of arrows and the relevant parser method?

**Pull request merged**